src/Microsoft.OData.Core/Json/JsonValueUtils.cs

```diff
@@ -35,7 +36,7 @@ internal static class JsonValueUtils
         /// <summary>
         /// The NumberFormatInfo used in OData Json format.
         /// </summary>
-        internal static readonly NumberFormatInfo ODataNumberFormatInfo = InitializeODataNumberFormatInfo();
+        internal static readonly NumberFormatInfo ODataNumberFormatInfo = JsonValueUtils.InitializeODataNumberFormatInfo();

```

Comment:
There's a bit of code churn from adding the JsonValueUtils namespace. For our coding style, we can leave it out if the function is within the same namespace.

Reply:
Is it ok to leave this for now? It's not hurting and makes it easy to distinguish between static and instance methods easily.

Reply:
I think that's fine. I don't think we strictly have any guidelines regarding keeping it or leaving it out.